lib/js/components/Buttons/IconButton/IconButton.vue

```diff
@@ -289,6 +292,7 @@ export default defineComponent({
 		WnlIcon,
 		WnlButton,
 	},
+	mixins: [hoverState],
```

Comment:
Let's not use mixins anymore, we can do better now. 🙏 
It's a good opportunity to talk with @pioziol123 about composability conventions, see https://github.com/bethinkpl/design-system/pull/424#discussion_r1796570861.